packages/clerk-js/src/ui/components/KeylessPrompt/index.tsx

```diff
@@ -252,111 +273,106 @@ const _KeylessPrompt = (_props: KeylessPromptProps) => {
             </p>
           </Flex>
 
-          {isExpanded && !claimed && (
-            <button
-              onClick={() => setIsExpanded(false)}
-              aria-label='Close'
-              type='button'
-              css={css`
-                cursor: pointer;
-                margin-left: 0.75rem;
-                color: #8c8c8c;
-                transition: color 130ms ease-out;
-                :hover {
-                  color: #eeeeee;
-                }
-                animation: show-button 200ms cubic-bezier(0.4, 0, 0, 1.1) forwards;
-
-                @keyframes show-button {
-                  from {
-                    transform: scaleX(0.9);
-                    opacity: 0;
-                  }
-                  to {
-                    transform: scaleX(1);
-                    opacity: 1;
-                  }
-                }
-              `}
-            >
-              <svg
-                width='1rem'
-                height='1rem'
-                viewBox='0 0 16 16'
-                fill='none'
-                aria-hidden
-                xmlns='http://www.w3.org/2000/svg'
-              >
-                <path
-                  d='M3.75 8H12.25'
-                  stroke='currentColor'
-                  strokeWidth='1.5'
-                  strokeLinecap='round'
-                  strokeLinejoin='round'
-                />
-              </svg>
-            </button>
-          )}
-        </Flex>
-
-        {isExpanded && (
-          <p
+          <svg
+            width='1rem'
+            height='1rem'
+            viewBox='0 0 16 16'
+            fill='none'
+            aria-hidden={!isExpanded || claimed}
+            aria-label='Minimize keyless overlay'
```

Comment:
```suggestion
            aria-hidden
```

The icon is only decorative so should be hidden all the time